src/tidysic.py

```diff
@@ -19,19 +19,67 @@ def guess_file_metadata(filename):
         if separator > 0:
             artist = filename[0:separator].lstrip()
             title = filename[separator + 2:len(filename)].lstrip()
-
-            return (artist, title)
+            
+            if guess_file_metadata.accept_all:
+                return (artist, title)
+            else:
+                # ask user what to do
+                message = [
+                    f"Guessed {artist}, {title} ",
```

Comment:
Could format artists and titles with colors for better readibility. Colors below are a random suggestion.
```suggestion
                    f"Guessed [blue]{artist}[/blue], [yellow]{title}[/yellow] ",
```